generate_workflow.py

```diff
@@ -43,23 +46,33 @@ def get_steps(build):
     name = get_job_name(build)
     return {
         sanitize_job_name(name): {
-            "runs-on": "ubuntu-20.04",
-            "needs": "kick_tuxsuite",
-            "name": name,
+            "runs-on":
+            "ubuntu-20.04",
+            "needs":
+            "kick_tuxsuite",
+            "name":
+            name,
```

Comment:
this change "hurts" but formatter knows best, I suppose...

Reply:
I can add `# yapf: disable` and `# yapf: enable` if we want to keep that the way it is for readability?

Reply:
sure, at the least we won't have to go look up the syntax for that every time we want to overrule the linter.